### PR TITLE
Move top level command under serve and add directory serve, refactor

### PR DESCRIPTION
**Pull request by Morishiri**

### CODE REVIEW

File: cmd/serve.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	lm "github.com/loophole/cli/internal/app/loophole/models"
+	"github.com/mitchellh/go-homedir"
+
+	"github.com/spf13/cobra"
+)
+
+var remoteEndpointSpecs lm.RemoteEndpointSpecs
+
+// accountCmd represents the account command
+var serveCmd = &cobra.Command{
```
Comment (Morishiri) [inline cmd/serve.go:15]: Option: Remove `serve` parent and move other commands right under top-level command

File: cmd/path.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+	"github.com/loophole/cli/internal/app/loophole"
+	lm "github.com/loophole/cli/internal/app/loophole/models"
+	"github.com/spf13/cobra"
+)
+
+var dirEndpointSpecs lm.LocalDirectorySpecs
+
+var dirCmd = &cobra.Command{
+	Use:   "dir <path>",
```
Comment (denny241) [inline cmd/path.go:14]: I think the filename should be consistent with the command name for clarity

File: internal/app/loophole/loophole.go [hunk 3]
```diff
@@ -122,279 +74,206 @@ func handleClient(client net.Conn, local net.Conn) {
...
 	}
 	if el := log.Debug(); el.Enabled() {
 		fmt.Println()
 		el.Msg("Listening on remote endpoint for HTTPS succeeded")
 	}
+	return listenerHTTPSOverSSH
+}
+
+func parsePublicKey(terminalState *terminal.State, identityFile string) (ssh.AuthMethod, ssh.PublicKey) {
+	communication.StartLoading("Obtaining public SSH key")
```
Comment (denny241) [inline internal/app/loophole/loophole.go:189]: This loader will display on top of the "Enter SSH Password:" prompt if the prompt has to be displayed.

File: cmd/path.go (hunk 2 above)
Comment (Morishiri) [reply cmd/path.go:14]: I updated that to be `path` and added the aliases `dir, directory`

File: internal/app/loophole/loophole.go (hunk 3 above)
Comment (Morishiri) [reply internal/app/loophole/loophole.go:189]: good catch

**Pull request merged**